game/i18n.hh

```diff
@@ -2,26 +2,74 @@
 
 #include <boost/locale.hpp>
 #include <iostream>
+
 #include "fs.hh"
+#include "configuration.hh"
 
 #define _(x) boost::locale::translate(x).str()
 #define translate_noop(x) x
 
 class TranslationEngine {
 public:
-	TranslationEngine(const char *package) {
+	TranslationEngine(const char *package) : m_package(package) {
+		auto lang = boost::locale::util::get_system_locale(true);
+
+		std::clog << "locale/debug: Checking if language is set in config." << std::endl;
+		if(config["game/language"].i() >= 0) {
+			lang = config["game/language"].getEnumStringValueAtIndex(config["game/language"].i());
+			std::clog << "locale/debug: Language in configuration found: " << lang << std::endl;
+		}
+
+		if (lang.empty()) lang = "en_US.UTF-8";
+		setLanguage(lang);
+	};
+	static bool enabled() {
```

Comment:
seems useless

Reply:
Not sure but will check it to make it sure :+1:

Reply:
Hmm for now i'll keep it in, can't really find a way to toggle between GETTEXT_FOUND or not which is what it should toggle